build/pgo-apiserver/Dockerfile

```diff
@@ -3,7 +3,7 @@ WORKDIR /go/src/github.com/percona/percona-postgresql-operator
 
 COPY . .
 
-RUN GOBIN="$(pwd -P)/hack/tools" ./hack/update-codegen.sh
+#RUN GOBIN="$(pwd -P)/hack/tools" ./hack/update-codegen.sh
```

Comment:
pls)

Reply:
yeah, forgot to remove completely https://github.com/percona/percona-postgresql-operator/pull/84/commits/644ebe394c00be8d5a4215ae15a29c1a6fe00027